src/include/btree.h

```diff
@@ -256,6 +270,8 @@ struct __wt_btree {
      * code.
      */
     WT_REF *evict_ref;                         /* Eviction thread's location */
+    WT_REF *evict_ref_saved;                   /* Eviction saved thread's location as an ID */
```

Comment:
ref is a very confusion term to be used here, as the purpose is not to use it as ref, in the sense, it will not be dereferenced. This might be confusing and some developers might use this to deference, which might be a difficult bug to catch, even there is a high chance that this will slip in code review

Reply:
Done